application/__init__.py

```
@@ -26,15 +26,22 @@
 from werkzeug.exceptions import HTTPException
 from werkzeug.serving import WSGIRequestHandler
 
+logger = logging.getLogger(__name__)
+
 try:
     with open('log_config.yaml', 'r') as config_file:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)